### DISCUSSION TITLE
How to hide a page from showing in search results

### DISCUSSION THREAD

**Discussion by kyle-TSH** [requarks/wiki#5811] [Help / Questions] [answered]
I would like to find a work-around to prevent a public page from showing in the search results. 

Currently our wiki has several page types that should not show in searches:

- Table of Contents pages (lists of multiple articles)
- Redirect pages
- Public articles that we only wish to share by link, but do not allow the user to browse these articles. 

Our redirect pages are themselves a work-around. I understand a proper redirect feature is in-progress: 
https://feedback.js.wiki/wiki/p/redirect-pages

and it is likely the proper redirect will mostly solve our issue. However, in the meantime we are creating pages with this script:

`<meta http-equiv="Refresh" content="0; url='[link to your page]'" />`

to redirect to a given page. However, all of these redirect pages show up in our search results. We have intentionally avoided using tags on these pages so they will not show in the tag browser. Can anyone think of a good way to suppress these from showing in the search results? 

Even after the proper redirect feature is implemented, we will still have public pages that should not show in search. Our wiki is a support knowledge base for our customers. We occasionally create support articles that we only wish to share by direct link. These articles must still be public-- it's too cumbersome for the user to have an account-- but we do not want them to show in the search results. 

Any ideas?

**Comment by NGPixel** [comment, marked as answer]
What you're asking is not possible at the moment in Wiki.js v2. It's however implemented in the upcoming v3. You'll be able to set per page whether it appears in the site tree and search results.

**Reply by kyle-TSH** [reply]
That's great to hear! I'm looking forward to v3. 

I'd still be interested in any hack-y work around in the meantime, if anyone has ideas. 

I found this old feature suggestion, where someone (I think it was you, same profile pic) suggested it was possible to unlist a page using Page Rules, but still be able to view the page by direct link:
https://feedback.js.wiki/wiki/p/ability-to-make-page-visible-only-by-link

Was this actually possible? Did something change?

Appreciate the help! Looking forward to the next version.

**Reply by NGPixel** [reply]
There's no workaround at the moment. The suggestion you linked no longer applies.
